docs/planning/frontend/src/pages/ReviewerPage.jsx

```
@@ -0,0 +1,250 @@
+                      type="text"
+                      className="input flex-1"
+                      placeholder="Optional context: 'for summer', 'date night', etc."
+                      value={suggestionContext}
+                      onChange={(e) => setSuggestionContext(e.target.value)}
+                    />
+                    <button
+                      onClick={() => suggestMutation.mutate()}
+                      disabled={suggestMutation.isPending}
+                      className="btn btn-primary"
+                    >
+                      {suggestMutation.isPending ? 'Thinking...' : 'Get Suggestions'}
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)